cli/admin/README.md

```diff
@@ -52,7 +52,7 @@ You can run the Pravega Admin CLI as follows:
 ```
 ./bin/pravega-admin
 ```
-You will se an output related to the default configuration parameters available at `conf/config.properties`
+You will see an output related to the default configuration parameters available at `conf/admin-cli.properties`. The values can be changed at the properties file (`conf/admin-cli.properties`), or use `config set property=...` to change the value.
```

Comment:
Listing the filename twice in the same short paragraph feels weird. Sentence could be rewritten so properties file reference is more clear and so filename is only listed once. The parenthetical expression below this line also feels out of place, incomplete or like a hold-over from a rough-draft, given the introduction of a second sentence.

Reply:
Consider pointing out that `config set property=...` only temporarily adjusts configs for the given command-line session only and that it doesn't make changes to the file from which defaults are loaded.